### ISSUE TITLE
Capture the freed space instead of calculate it

### ISSUE THREAD

**Issue by Tobias-B-Besemer** [bleachbit/wishlist#21]
As we wrote about using external tools to free up space, too, I think it would be a good idea (a accurate solution) to capture the free space (no calculate it) of drive C: before cleaning, after cleaning and then display it (and calculate/show the difference).

A displaying at the end can then maybe look like this:

You had before cleaning 80GB free space on drive C:.
Now, after cleaning, you have 95GB free space on drive C:.
That means BleachBit freed up 15GB free space for you on drive C:! Congratulations!

**Comment by az0** [comment]
Yes, this would be nice. It could be a little inaccurate if other processes are changing files, but this is not a critical issue
